spec/features/cases/offender_sar/case_creating_spec.rb

```diff
@@ -10,45 +10,109 @@
     cases_page.load
   end
 
-  scenario 'creating a case that does not need clearance', js: true do
+  scenario '1 Third party requests data to be sent to the data subject' do
```

Comment:
This is great - I like the cucumber style method names - it's not in the same style as the rest of the features specs here but happy to adopt this style 

Just wondering if the scenario names properly reflect what's happening? 

I think the `and_fill_in_requester_details_page` method defaults to subject requesting the data

Reply:
Have updated the spec to reflect this 👍